src/app/service-code-way/service-code-way.component.ts

```diff
@@ -21,7 +21,7 @@ export class ServiceCodeWayComponent implements OnDestroy, OnInit {
     allowedTypes: 'image/*,video/*',
     url: `${environment.api}/upload?uploadType=uploadx`,
     token: 'someToken',
-    autoUpload: false,
+    autoUpload: true,
```

Comment:
maybe `autoUpload: false`?
because it conflicts with
`  this.uploadService.control({
          action: 'upload',
          itemOptions: cfg,
          uploadId: item.uploadId
        });
`

Reply:
fixed